<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(100, 100, 100);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>220</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(230, 230, 230);</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>


   <widget class="QPushButton" name="buttunnum0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>369</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>


   <widget class="QPushButton" name="buttunnum1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   
   <widget class="QPushButton" name="buttunnum2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>


   <widget class="QPushButton" name="buttunnum3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>


   <widget class="QPushButton" name="buttunnum4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>


   <widget class="QPushButton" name="buttunnum5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>


   <widget class="QPushButton" name="buttunnum6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>


   <widget class="QPushButton" name="buttunnum7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>


   <widget class="QPushButton" name="buttunnum8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>


   <widget class="QPushButton" name="buttunnum9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>

   
   <widget class="QPushButton" name="minbuttun">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 165, 0);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>



   <widget class="QPushButton" name="enterbuttun">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0); 
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>


   <widget class="QPushButton" name="cbuttun">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>



   <widget class="QPushButton" name="dotbuttun">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 80, 80);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>


   <widget class="QPushButton" name="sumbuttun">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 165, 0);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>



   <widget class="QPushButton" name="prcbuttun">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 165, 0);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>


   <widget class="QPushButton" name="mulbuttun">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 165, 0);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>




   <widget class="QPushButton" name="divisionbuttun">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 165, 0);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>





  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>